backend/models/client.py

```diff
@@ -28,6 +28,10 @@ def serialize(self) -> dict:
 class Client_Properties(SQLModel, table=True):
     client_id: int = Field(foreign_key="client.id", primary_key=True)
     name: str = Field(
-        sa_column=Column(String, unique=True),
+        sa_column=Column(String),
     )
     client: "Client" = Relationship(back_populates="properties")
+
+    __table_args__ = (
+        UniqueConstraint("client_id", "name", name="unique_client_name"),
```

Comment:
Is this a constraint over user id or user id + name? It should only be on user id, right?